charts/src/main/java/hu/ma/charts/legend/VerticalLegend.kt

```diff
@@ -16,31 +13,26 @@ import androidx.compose.ui.util.fastForEachIndexed
 import hu.ma.charts.internal.DefaultText
 
 @Composable
-fun RowScope.DrawVerticalLegend(
+fun DrawVerticalLegend(
   legendEntries: List<LegendEntry>,
   text: @Composable (entry: LegendEntry) -> Unit = {
     DefaultText(text = it.text)
   },
 ) {
-  Column(
-    modifier = Modifier.weight(1f),
```

Comment:
Same here

Reply:
Will use a separate legend for LineChart. thought I had verified it was still working for the other charts, but I have probably overlooked something :)